src/main/java/de/muenchen/isi/infrastructure/entity/stammdaten/FoerdermixStamm.java

```diff
@@ -28,6 +29,6 @@ public class FoerdermixStamm extends BaseEntity {
     @Column(length = 80, nullable = false)
     private String bezeichnung;
 
-    @Embedded
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
CascadeType.ALL:

Wird der Fördermixstamm gelöscht wird somit auch der Fördermix in der Fördermixtabelle gelöscht. Die Baurate verliert somit seine referenzierte Entität.

Im umgekehrten Fall beim Löschen einer Baurate verliert der Fördermixstamm  ebenso seine referenzierte Entität.